docs/docs/guides/tools/comparison.md

```diff
@@ -0,0 +1,113 @@
+2. Click the six dots to the left of the ID.
+   ![Setting the order.](imgs/comparison-2objs-reorder.png)
+3. Drag the ID to the left or the right, depending on which object was selected. 
+4. Place the ID in the desired ordering. The UI refreshes with an updated comparison ordering.
+
+### Change numeric diff display format 
+
+For numeric values such as `completion_tokens` and `total_tokens`, you can view the diff as either an integer or a percentage. To change a numeric diff's display format, do the following:
+
+1. In the Comparison table, find the numeric value that you want to update the diff display format for.
+    ![A numeric value displayed as an integer.](imgs/comparison-2objs-numericdiffformat.png)
+2. Click the diff value. The format automatically updates to either an integer or a percentage.
+    ![A numeric value updated to a percentage.](imgs/comparison-2objs-numericdiffformat-updated.png)
+
+### Compare with baseline or previous
+
+> This option is only available when comparing 3 or more objects.
+> You can also [set](#set-a-baseline) or [remove an existing baseline by clicking the 3 dots to the right of the ID](#remove-a-baseline).
+
+To perform a baseline comparison with 3 or more objects, do the following:
+
+1. In the right hand corner of the Comparison view, click the dropdown. Depending on your current view configuration, the dropdown is either titled **Compare with previous** or **Compare with baseline**.
+2. Depending on your current view configuration, select either **Compare with previous** or **Compare with baseline**.
+   - **Compare with baseline**: Sets the leftmost object as the baseline. The table updates so that the leftmost column is the baseline.
+   -  **Compare with previous**: No object is set as baseline.
+
+### Compare a subset of objects
+
+> This option is only available when comparing 3 or more objects.
+
+When comparing 3 or more objects, you can compare a single object to a previous object or baseline. This changes the Comparison table view so that the view is identical to a two-object comparison. To compare a subset of objects, do the following:
+
+1. In the Comparison view topbar, find the ID that you want to compare to previous or baseline. 
+2. To select the item, click the ID. The UI refreshes with a two-way comparison table.
+    ![Comparing a subset of objects.](imgs/comparsion-7objs-diffonly-subset.png)
+
+To reset the view so that the first 6 objects selected for comparison are displayed in the table, click the ID again.
+
+### Remove an object from comparsion
+
+> This option is only available when comparing 3 or more objects.
+
+To remove an object from comparison, do the following:
+
+1. In the Comparison view topbar, find the object that you want to remove from comparison.
+2. Click the three dots to the right of the ID.
+3. In the dropdown, select **Remove object from comparison**. The UI refreshes with an updated that no longer includes the removed object.
```

Comment:
Missing a word/words after "updated"

Reply:
Addressed in https://github.com/wandb/weave/pull/3124/commits/65d43eb765868268559f8573347bdcc4a3302dcd